src/forgot-password/APIFailureMessage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Alert } from '@edx/paragon';
+
+const APIFailureMessage = () => (
+  <div className="d-flex justify-content-center m-4">
+    <div className="d-flex flex-column">
+      <div className="text-center mw-500">
```

Comment:
Also, no need to wrap this message inside all these divs if it is going to appear at the top of the form.